edx_exams/apps/api/v1/urls.py

```diff
@@ -2,11 +2,16 @@
 
 from django.urls import re_path
 
-from edx_exams.apps.api.v1.views import CourseExamsView
+from edx_exams.apps.api.v1.views import CourseExamConfigurationsView, CourseExamsView
 from edx_exams.apps.core.constants import COURSE_ID_PATTERN
 
 app_name = 'v1'
 
 urlpatterns = [
-    re_path(fr'exams/course_id/{COURSE_ID_PATTERN}', CourseExamsView.as_view(), name='exams-course_exams'),
+    re_path(fr'exams/course_id/{COURSE_ID_PATTERN}',
+            CourseExamsView.as_view(),
+            name='exams-course_exams'),
+    re_path(fr'configs/course_id/{COURSE_ID_PATTERN}',
```

Comment:
NIT, but 'configs' to me reads as plural. I'd just go with configuration or config

Reply:
Updated to config!

Reply:
@Zacharis278 I was just looking at this and it seems to indicate that it should be plural: https://openedx.atlassian.net/wiki/spaces/AC/pages/18350757/edX+REST+API+Conventions
But maybe I'm misunderstanding?

Reply:
I'm reading it the same way. Plural it is, then! Thanks for double checking that.

Reply:
huh yeah 'configs' just struck me as odd at first glace since there is only one per course but that doc is definitely correct. Guess it should be plural. I rescind my comment!

Reply:
Sweet! Thanks for confirming @MichaelRoytman and @Zacharis278! Will update.